src/bin/ipc_latency.rs

```diff
@@ -43,30 +44,75 @@ impl portus::serialize::AsRawMsg for TimeMsg {
         Ok(TimeMsg(time::Timespec::new(sec, nsec)))
     }
 }
+#[derive(Debug)]
+struct DetTimeMsg(time::Timespec, time::Timespec);
```

Comment:
What's "Det"?

Reply:
It's supposed to be short for "detailed" but I can see how that's a bad name.